File: NotificationSystem/NotificationSystem.Data/Classes/clsStatusList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace NotificationSystem.NotificationSystem.Data.Classes
{
	public class clsStatusList
	{

		private string m_Status = string.Empty;
		private int m_TroubleTicketNo = 0;

        public string Status
        {
            get
            {
                return m_Status;
            }
            set
            {
                m_Status = value;
            }
        }

        public int TroubleTicketNo
        {
            get
            {
                return m_TroubleTicketNo;
            }
            set
            {
                m_TroubleTicketNo = value;
            }
        }

    }
}

File: NotificationSystem/NotificationSystem.Data/Classes/clsStatusList.vb

Namespace NotificationSystem.NotificationSystem.Data.Classes
    Public Class clsStatusList

        Private m_Status As String = String.Empty
        Private m_TroubleTicketNo As Integer = 0

        Public Property Status As String
            Get
                Return m_Status
            End Get
            Set(value As String)
                m_Status = value
            End Set
        End Property

        Public Property TroubleTicketNo As Integer
            Get
                Return m_TroubleTicketNo
            End Get
            Set(value As Integer)
                m_TroubleTicketNo = value
            End Set
        End Property

    End Class
End Namespace
